CMakeLists.txt

```
@@ -2,7 +2,7 @@ cmake_policy(SET CMP0048 NEW)
 project(S2E_FOR_C2A_CORE
   LANGUAGES CXX
   DESCRIPTION "S2E_FOR_C2A_CORE"
-  VERSION 1.0
+  VERSION 2.0
```

書くなら patch version まで書いておいた方がよさそう（少なくともフォーマット上は semver としていきたいので）
```suggestion
  VERSION 2.0.0
```

---

修正しました。